tests/unit/services/shieldPersistenceService.test.js

```
@@ -0,0 +1,638 @@
+  eq: jest.fn().mockReturnThis(),
+  gte: jest.fn().mockReturnThis(),
+  lte: jest.fn().mockReturnThis(),
+  lt: jest.fn().mockReturnThis(),
+  is: jest.fn().mockReturnThis(),
+  not: jest.fn().mockReturnThis(),
+  order: jest.fn().mockReturnThis(),
+  limit: jest.fn().mockReturnThis(),
+  range: jest.fn().mockReturnThis(),
+  single: jest.fn(),
+  rpc: jest.fn(),
+  then: jest.fn().mockImplementation(resolve => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)